src/screens/WebView/TermsAndCondition.js

```diff
@@ -4,6 +4,7 @@ import {ActivityIndicator, StyleSheet, View, SafeAreaView} from 'react-native';
 import {WebView} from 'react-native-webview';
 import Header from '../../components/Header';
 import { withInteractionsManaged } from '../../components/WithInteractionManaged';
+import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `COLORS` from the `theme` module is a good change as it centralizes the color definitions in one place. This makes the code more maintainable and consistent.
```diff
+ import {COLORS} from '../../utils/theme';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->